src/main/java/mate/academy/spring/model/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,22 +1,15 @@
 package mate.academy.spring.model.dto.request;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
do you need both these annotations?

Reply:
i can't use the builder without them

Reply:
constructor CinemaHallRequestDto in class mate.academy.spring.model.dto.request.CinemaHallRequestDto cannot be applied to given types;

Reply:
must use both of them